### ISSUE TITLE
Wonder if it may be possible to update so you can update invertor settings

### ISSUE THREAD

**Issue by ram1t** [richardlevy/solis-cloud-scraper#3]
Wonder if its possible to update this so that not only  can it scrape from sols-cloud but send through settings to invertor control on solis cloud such as charge/discharge times?

**Comment by richardlevy** [comment]
Is that possible from the solis cloud website?Sent from my iPhoneOn 26 Feb 2023, at 13:13, ram1t ***@***.***> wrote:﻿
Wonder if its possible to update this so that not only  can it scrape from sols-cloud but send through settings to invertor control on solis cloud such as charge/discharge times?

**Comment by ram1t** [comment]
Yes it is possible now - you have to request it via a ticket from solis (look on their website) and then the invertor control option appears on app and on website

**Comment by yetkinergun** [comment]
This is exciting! FAQ link [here](https://solis-service.solisinverters.com/support/solutions/articles/44002373796-inverter-remote-control-application). I've opened a ticket and will see what we can control.

**Comment by chiastic-security** [comment]
Is there any progress on this? It would be amazing to be able to do this.

My most pressing concern is that the inverter clock seems to drift, and that means that my charging time gets out of sync, so I want to set up a cron job to set the clock once per day.

(It's a bit depressing having to do this... you'd think that it could do it for itself.)

**Comment by Webreaper** [comment]
I'm interested in this too. I've got remote control enabled via the soliscloud app/site, so it seems like it would be trivial to extend this script to drive the inverter control.

My intent is to just use screen scraping to toggle the charging ampage between 0 and 95A, to disable/enable charging overnight. If that can be scripted, then it could be called by an HA integration triggered by solcast - allowing the key goal of only charging the battery the night before cloudy days where PV yield is expected to be poor.
